src/main/java/appeng/container/implementations/ContainerInterfaceTerminal.java

```diff
@@ -56,13 +57,13 @@ public final class ContainerInterfaceTerminal extends AEBaseContainer {
     private PacketIfaceTermUpdate dirty;
     private boolean isDirty;
     private IGrid grid;
-    private PartInterfaceTerminal partRef;
+    private IActionHost anchor;
     private boolean wasOff;
 
-    public ContainerInterfaceTerminal(final InventoryPlayer ip, final PartInterfaceTerminal anchor) {
+    public ContainerInterfaceTerminal(final InventoryPlayer ip, final IActionHost anchor) {
```

Comment:
Where is this being used?

Reply:
InterfaceWireless container use it as delegate container